### PR TITLE
MVP-349 Capabilities Home Screen

### PR DESCRIPTION
**Pull request by jestanoff**
- Creates new django template and url for /learn landing page
- Adds files some python generated files to `.gitignore`
- Adds prop-types npm package for react components
- Moves Enzyme setup to `react-components/setupJest.js`, so these doesn't need to be added for each unit test file

 - [X] Change has a jira ticket that has the correct status.
 - [X] Changelog entry added.

### CODE REVIEW

**Comment by Trunk89** [comment]
FE code looks good to me, but would be good if it was reviewed by Edie or Rich to make sure it follows the project standards.

File: tests/unit/learn/test_views.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import pytest
+
+
+@pytest.mark.django_db
+def test_learn_landing_page(client, exportplan_homepage):
+    response = client.get('/learn/')
```
Comment (sdonk) [inline tests/unit/learn/test_views.py]: can you use reverse?

File: tests/unit/learn/test_models.py [hunk 2]
```diff
@@ -56,3 +56,13 @@ def test_topic_view(client, domestic_homepage, user, domestic_site):
 
     response = client.get(topic.url)
     assert response.status_code == 200
+
+@pytest.mark.django_db
+def _view(client, domestic_homepage, user, domestic_site):
```
Comment (sdonk) [inline tests/unit/learn/test_models.py]: this test won't run, it doesn't start with `test`

File: learn/models.py [hunk 3]
```diff
@@ -87,3 +87,6 @@ class LessonViewHit(TimeStampedModel):
     class Meta:
         ordering = ['lesson__pk']
         unique_together = ['lesson', 'topic', 'sso_id']
+
+class LandingPage(Page):
+    parent_page_types = ['domestic.DomesticHomePage']
```
Comment (sdonk) [inline learn/models.py]: can you please add a test to check the hierarchy, something like this https://github.com/uktrade/great-cms/blob/develop/tests/unit/domestic/test_models.py#L12

File: tests/unit/learn/factories.py [hunk 4]
```diff
@@ -26,3 +26,14 @@ class LessonPageFactory(wagtail_factories.PageFactory):
     class Meta:
         model = models.LessonPage
         django_get_or_create = ['slug', 'parent']
+
+class LandingPageFactory(wagtail_factories.PageFactory):
+
+    title = 'landing page'
+    live = True
+    slug = ''
```
Comment (sdonk) [inline tests/unit/learn/factories.py]: Not sure it's correct to create a page with an empty slug 🤔

Comment (jestanoff) [reply tests/unit/learn/factories.py]: Alright, silly me. Removed!

File: learn/urls.py [hunk 5]
```diff
@@ -0,0 +1,9 @@
+from django.urls import path
+
+from learn import views
+
+app_name = 'learn'
+
+urlpatterns = [
+    path('', views.LearnLandingPageView.as_view(), name='index'),
```
Comment (sdonk) [inline learn/urls.py]: You can remove this, wagtail will take care of the routing

File: config/urls.py [hunk 6]
```diff
@@ -21,6 +22,7 @@
     path('sso/', include(sso.urls)),
     path('', include(core.urls, namespace='core')),
     path('export-plan/', include(exportplan.urls)),
+    path('learn/', include(learn.urls)),
```
Comment (sdonk) [inline config/urls.py]: do we want to let wagtail do the routing?

File: core/templates/core/header.html [hunk 7]
```diff
@@ -7,7 +7,7 @@
     </a>
     <div class="great-mvp-header-links">
       <ul id="services-area" class="services-area">
-          <li><a id="header-link-learning" href="#">Learning</a></li>
+          <li><a id="header-link-learning" href="/learn/">Learning</a></li>
```
Comment (sdonk) [inline core/templates/core/header.html]: can you please use  this https://docs.wagtail.io/en/v2.8/topics/writing_templates.html#internal-links-tag?

File: learn/templates/learn/learn_page.html [hunk 8]
```diff
@@ -0,0 +1,11 @@
+{% extends "core/base.html" %}
+
+{% block content %}
+    <section id="learn-root">
+        <div class="learn-wrapper">
+            <h1 class="h1">Start exporting today</h1>
+            <p class="paragraph">Start your exporting journey today, and in no time, you’ll have your realistic business plan for selling overseas.</p>
+            <a class="button" href="/learn/learn-to-export-step-1">Learn how to export</a>
```
Comment (sdonk) [inline learn/templates/learn/learn_page.html]: https://docs.wagtail.io/en/v2.8/topics/writing_templates.html#internal-links-tag

Comment (jestanoff) [reply learn/templates/learn/learn_page.html]: That url doesn't exist yet, it is going to be implemented in another ticket

**Pull request merged**